database.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+$db = new PDO ('mysql:host=db; dbname=kimberley-collection', 'root', 'password');
+$db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+$query = $db->prepare("SELECT `name`,`main-colour`,`category`,`material`,`size`,`heavyness`,`img-url` FROM `kimberley-collection`;");
+
+$query->execute();
+
+$outputs = $query->fetchAll();
+
+//echo '<pre>';
+//var_dump($outputs);
+//echo '</pre>';
+
+    foreach ($outputs as $key => $output) {
```

Comment:
All looks good! Only thing I think is even worth a mention at the moment is that using $key in this loop is unnecessary right now as it's not being utilised in the loop. It would work just the same without it and just having "$outputs as $output"

Reply:
great thank you!!